mongodbatlas/resource_mongodbatlas_global_cluster_config.go

```diff
@@ -348,3 +379,36 @@ func addManagedNamespaces(ctx context.Context, conn *matlas.Client, add []interf
 
 	return nil
 }
+
+func expandCustomZoneMapping(tfMap map[string]interface{}) *matlas.CustomZoneMapping {
```

Comment:
[nit] naming, functions that return a new instance of a struct tend to start with `new`
```suggestion
func newCustomZoneMapping(tfMap map[string]interface{}) *matlas.CustomZoneMapping {
```